packages/eslint-plugin/src/rules/no-empty-interface.ts

```diff
@@ -76,26 +77,41 @@ export default util.createRule<Options, MessageIds>({
 
             // Check if interface is within ambient declaration
             let useAutoFix = true;
+            let hasSuggest = true;
```

Comment:
Nit: I found it confusing how the previous variable starts with _use_ but this starts with _has_, even though they're pretty much the same thing. Can we make the names more similar?

```suggestion
            let useSuggestion = true;
```

Note: after applying the refactor I suggest later in the file, the variable might be unnecessary.